### DISCUSSION TITLE
Way to detect missing/misspelled handlers

### DISCUSSION THREAD

**Discussion by iwane-pl** [ansible/ansible-lint#3906] [Ideas]
Is there a way for `ansible-lint` to detect handlers that are missing/misspelled? 

For example, we have roles that use lower-case naming for tasks. After `ansible-lint --fix`, names are Title-cased automatically, but `notify` blocks are not. This makes playbooks fail due to handlers being not found.

If `ansible-lint` would detect it, we could catch those errors on pre-commit level instead of tracing failures in Jenkins :)

**Comment by sbrocket** [comment]
Sad to see that this suggestion has sat here for years as I think it would be a very valuable check! I checked today on ansible-lint 26.3.0 and it does not seem to catch this yet.

It perhaps is more "static analysis" than lint, but ansible-lint still seems like the best fit for it short of there being sufficient other cause for a separate bug/error analysis tool.
